.github/workflows/nox.yml

```diff
@@ -49,6 +49,8 @@ jobs:
           python-versions: "3.9, 3.10, 3.11"
       - name: nox -e test
         run: |
-          OTHER_ANTSIBULL_MODE=git nox -v -e test -- --cov-report=xml
+          OTHER_ANTSIBULL_MODE=git nox -v -e test -- --cov-report=xml --cov-append
```

Comment:
Shoudn't this be added in the noxfile itself?

Reply:
It's probably unwanted when running locally. It's safe to use in CI, because it's a clean environment with no existing cov data.